app/views/incomeSources/cease/CeaseIncomeSources.scala.html

```diff
@@ -149,61 +143,53 @@
         @p(id = Some("foreign-property-p1")) {
             @getMessage("foreign-property.p1") }
 
-        @govukTable(Table(
-            head = Some(List(
-                HeadCell(
-                    content = Text(getMessage("table-head.date-started")),
-                    attributes = Map("id" -> "table-head-date-started-foreign")),
-                HeadCell(
-                    content = Empty,
-                    attributes = Map("id" -> "empty3"))
-            )),
-            rows = Seq(Seq(
-                TableRow(
-                    content = Text(sources.foreignProperty.map(_.tradingStartDate.toLongDate).get),
-                    attributes = Map("id" -> "table-row-trading-start-date-foreign")),
-                TableRow(
-                    content = HtmlContent(
-                        s"""<dd class="govuk-summary-list__actions">
-                                <a class="govuk-link" href='$ceaseForeignPropertyURL'>
-                                    ${getMessage("cease")}<span class="govuk-visually-hidden">cease</span>
-                                </a>
-                            </dd>"""
-                    )
-                )
-            )),
-            attributes = Map("id" -> "foreign-property-table")
-        ))
+        <table class="govuk-table govuk-!-margin-bottom-3" id="foreign-property-table">
+            <thead class="govuk-table__head">
+                <tr class="govuk-table__row">
+                    <th class="govuk-table__header" scope="col" id="table-head-date-started-foreign">@getMessage("table-head.date-started")</th>
+                    <td class="govuk-table__header" id="empty2"></td>
+                </tr>
+            </thead>
+            <tbody>
+                <tr class="govuk-table__row">
+                    <td class="govuk-table__cell" id="table-row-trading-start-date-foreign">@sources.foreignProperty.map(_.tradingStartDate.toLongDate).get</td>
```

Comment:
same issue as above, we should properly handle scenario when foreignProperty is None